portfolio/src/main/webapp/script.js

```diff
@@ -124,3 +110,21 @@ function createListElement(text) {
   liElement.textContent = text;
   return liElement;
 }
+
+/**
+ * Fetches a list of messages and adds it to the page.
+ */
+async function displayComments() {
+    const response = await fetch('/list-comments');
+    const json = await response.json();
+
+    const ulElement = document.getElementById('comments-list');
+
+    for (comment of json) {
+        ulElement.appendChild(createListElement(comment));
+    }
+
+    if (ulElement.textContent === '') {
```

Comment:
Could we use the `json` or `response` to check if there are no comments yet? this is a more reliable way of making this check.

Since these operations are asynchronous, and the response could take a bit to get back to us, checking for the `textContent` on the document (synchronous), could happen first. This would result in having "No comments yet." show up, followed by the comments getting populated.